pages/demo.html

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+title: Online demo
```

Comment:
In this page add an info alert https://getbootstrap.com/docs/4.0/components/alerts/ saying:
"The demo refers to the version 1.0.0-beta. We are currently in the process of the complete rewrite of HospitalRun preparing it for the upcoming version 2.0.0. Follow the development on [link to the github mono-repo]"